spyder/plugins/editor/widgets/tests/test_warnings.py

```diff
@@ -36,8 +37,20 @@ def test_ignore_warnings(qtbot, lsp_codeeditor):
 
     CONF.set('lsp-server', 'pydocstyle/ignore', 'D100')
     CONF.set('lsp-server', 'pycodestyle/ignore', 'E261')
+
+    # After this call the manager needs to be reinitialized
     manager.update_server_list()
-    qtbot.wait(2000)
+
+    test_ci = os.environ.get('CI_PYTEST', os.environ.get('CI', None))
```

Comment:
@ccordoba12 @andfoy I really do not know why I need this locally :-| 🤷‍♂

Reply:
There's not a single instance of `CI_PYTEST` in our codebase, so I don't understand where you got that name. So I think you should remove this and simply test for the `CI` variable below.

Reply:
I was looking for CI variables and found this -> https://github.com/spyder-ide/spyder/blob/master/.travis.yml#L9

Reply:
Ok, I see. I introduced that variable to share the same bash scripts between Travis and Circle but now it's not needed, so you can remove it.